src/overlays/actors/ovl_En_Slime/z_en_slime.h

```diff
@@ -10,7 +17,32 @@ typedef void (*EnSlimeActionFunc)(struct EnSlime*, PlayState*);
 typedef struct EnSlime {
     /* 0x000 */ Actor actor;
     /* 0x144 */ EnSlimeActionFunc actionFunc;
-    /* 0x148 */ char unk_148[0xC0];
+    /* 0x148 */ u8 iceBlockTimer;
+    /* 0x149 */ u8 eyeTexIndex;
+    /* 0x14A */ u8 damageEffectType;
+    /* 0x14C */ s16 timer;
+    /* 0x14E */ s16 idleRotY;
+    /* 0x150 */ s16 respawnRotY;
+    /* 0x152 */ s16 respawnTime;
+    /* 0x154 */ union { 
+                    Vec3f iceBlockSnapPos;
+                    Vec3f wobbleRot;
+                };
+    /* 0x160 */ TexturePtr dropObjectTex;
+    /* 0x164 */ f32 effectAlpha;
+    /* 0x168 */ f32 effectScale;
+    /* 0x16C */ f32 frozenSteamScale;
+    /* 0x170 */ f32 distLimit;
+    /* 0x174 */ Vec3f respawnScale;
+    /* 0x180 */ Vec3f limbPos[EN_SLIME_LIMBPOS_COUNT];
+    /* 0x1BC */ ColliderCylinder collider;
 } EnSlime; // size = 0x208
 
+typedef enum EnSlimeType {
+    /* 0 */ EN_SLIME_TYPE_BLUE,
+    /* 1 */ EN_SLIME_TYPE_GREEN,
+    /* 2 */ EN_SLIME_TYPE_YELLOW,
+    /* 3 */ EN_SLIME_TYPE_RED,
+} EnSlimeType;
```

Comment:
Yeah, I think that calling them "color" instead of "type" might make more sense. It's not that type is *wrong* or anything, just that the type *is* the color.

Reply:
It is more than color tho, non-blue drop different items and blue spawns an iceblock if hit with ice arrows, color is arguably just a visual indicator of type, not just a cosmetic.

Reply:
I swapped it to "COLOR", should I switch it back to "TYPE" then?

Reply:
It's probably fine to leave it as color if you already changed it

Reply:
Yeah, I suppose that's fair. I still think "color" is fine, because as I said, the type *is* the color, but you could call it "type" too. Feel free to name it type *or* color, and I'll be happy with it.